client/src/reducers/patientCreation.ts

```diff
@@ -31,7 +31,7 @@ const reducer = (
   switch (action.type) {
     case actions.CREATE_PATIENT_SUCCEEDED:
     case actions.CREATE_PATIENT_FETUS_SUCCEEDED: {
-      draft.patient = action.payload.patient;
+      draft.patient = action.payload.patientFetus ? action.payload.patientFetus : action.payload.patient;
```

Comment:
You get here from `CREATE_PATIENT_FETUS_SUCCEEDED`, it means `action.payload.patientFetus` will always be defined (otherwise it would be `CREATE_PATIENT_FETUS_FAILED`).
Therefore, you could simply have `draft.patient = action.payload.patientFetus;` in this case